Documentation/ExtensionArchitecture/Tutorials/Tea/Model.rst

```diff
@@ -0,0 +1,298 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    Tutorial Tea; Model
+..  _extbase_tutorial_tea_model:
+
+===================
+Model: a bag of tea
+===================
+
+The model in this example is kept quite basic: Each tea can have a title,
+a description and optionally an image attached.
+
+..  uml::
+
+    class Tea {
+        title
+        ==
+        description
+        ==
+        image
+    }
+
+The title and description are strings, the image is stored as relation
+to the class :php:`\TYPO3\CMS\Extbase\Domain\Model\FileReference` provided
+by Extbase as model.
+
+
+..  _extbase_tutorial_tea_model_database:
+
+The database model
+==================
+
+Let us translate this into SQL and store the scheme in a file called
+:file:`ext_tables.sql`:
+
+..  include:: /CodeSnippets/Tutorials/Tea/ExtTablesSql.rst.txt
+
+The image is stored as an integer. However the field :sql:`image` in the
+database does not contains a reference to the image in form of an identifier.
+
+The field :sql:`image` stores how many images are attached. A separate table,
+a so called MM table stores the actual relationship. See about the definition
+of this field here: :ref:`extbase_tutorial_tea_model_columns_image`.
```

Comment:
This is a good point. The fact that the image field stores the number of references is also an optimization, because if there are no images, we don't have to look for them in a separate table.

Since you're saying that there is something called an MM table, you might want to say what MM stands for too. Otherwise, it's a little half-way to an explanation.

```suggestion
The field :sql:`image` keeps track of the number of attached images. A separate table,
a so-called MM table, stores the actual relationship. Read about the definition
of this field here: :ref:`extbase_tutorial_tea_model_columns_image`.
```